### PR TITLE
The next bunch of flags and fixes

### PR DESCRIPTION
**Pull request by nickkolok**
It's going to close `crucial` milestone.

### CODE REVIEW

File: ipt_connect/IPTdev/views.py [hunk 1]
```diff
@@ -387,9 +387,17 @@ def physics_fight_detail(request, pfid):
...
 			meanroundsgrades.append(meangrades)
 
-		for team in summary_grades:
-			summary_grades[team].append(sum(summary_grades[team]))
-		summary_grades = sorted(summary_grades.items(), key=lambda x: x[1][4], reverse=True)
+		try:
+			for team in summary_grades:
+				summary_grades[team].append(sum(summary_grades[team]))
+			summary_grades = sorted(summary_grades.items(), key=lambda x: x[1][4], reverse=True)
+		except:
```
Comment (vbonvin) [inline ipt_connect/IPTdev/views.py]: In which cases would the above fail?

File: ipt_connect/IPTdev/views.py [hunk 2]
```diff
@@ -389,7 +389,7 @@ def physics_fight_detail(request, pfid):
 
 		try:
 			for team in summary_grades:
-				summary_grades[team].append(sum(summary_grades[team]))
+				summary_grades[team].append(sum(summary_grades[team][1:]))
```
Comment (vbonvin) [inline ipt_connect/IPTdev/views.py]: I'm not sure to understand what was the problem here from your commit message. Could you detail a little bit more?

Also, if this is a bug, do you think it was there for long?

File: ipt_connect/IPTdev/views.py (hunk 1 above)
Comment (nickkolok) [reply ipt_connect/IPTdev/views.py]: `x[1][4]` can be undefined, e.g. for 4-fight.

File: ipt_connect/IPTdev/templates/IPTdev/ranking.html [hunk 3]
```diff
@@ -55,9 +55,11 @@ <h1>{% trans 'General Ranking' %}</h1>
...
 
-    <div>
-        <center>
-        <h3><a href="{% url 'IPTdev:poolranking' %}">{% trans 'Pools Ranking' %}</a></h3>
-        </center>
-    </div>
+    {% if params.enable_pools %}
+		<div>
+			<center>
+			<h3><a href="{% url 'IPTdev:poolranking' %}">{% trans 'Pools Ranking' %}</a></h3>
```
[the remark this answers is not in the document]
Comment (nickkolok) [reply ipt_connect/IPTdev/templates/IPTdev/ranking.html]: Yes, it does using `sed`: https://github.com/IPTnet/ipt_connect/blob/dev/ipt_connect/clone-instance.sh#L17
However, I've opened #78 to unhardcode this.

File: ipt_connect/IPTdev/views.py (hunk 2 above)
Comment (nickkolok) [reply ipt_connect/IPTdev/views.py]: The problem was that the reporter's multiplier was added to the sum of points.
This bug appeared tne same time with the summary tables, i.e. after IPT2018, so it affected no tournaments.

**Pull request merged**